mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -430,3 +430,16 @@ def test_io_set_raw_2021():
     assert "EEG" not in io.loadmat(raw_fname_2021)
     _test_raw_reader(reader=read_raw_eeglab, input_fname=raw_fname_2021,
                      test_preloading=False, preload=True)
+
+
+@testing.requires_testing_data
+def test_read_single_epoch():
+    """Test reading a single epoch/trial set file as Epochs"""
+    raw = read_raw_eeglab(raw_fname_mat, preload=True)
+    epochs = read_epochs_eeglab(raw_fname_mat)
```

Comment:
why are you using read_epochs_eeglab to read a raw continuous file?

Reply:
This is the simplest way I can find to pretend there's a single-epoch SET file, since the purpose is only to test if `read_epochs` works with single-epoch data.